src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingTownHall.java

```diff
@@ -113,10 +113,6 @@ public void serializeToView(@NotNull final PacketBuffer buf)
     @Override
     public int getClaimRadius(final int newLevel)
     {
-        if (newLevel + 1 < MineColonies.getConfig().getCommon().minTownHallPadding.get())
```

Comment:
Why was this removed?

Reply:
ah because we're always claiming the initial radius when creating the colony now